api/cpp/include/slint_platform.h

```diff
@@ -61,6 +61,9 @@ class AbstractWindowAdapter
     /// do that in the next iteration of the event loop, or in a callback from the window manager.
     virtual void request_redraw() const { }
 
+    /// Returns the actual physical size of the window
+    virtual slint::PhysicalSize physical_size() const = 0;
```

Comment:
The reason I haven't called this `size` is because we use the multiple inheritance trick with platform_qt and that conflict with QWindow::size who is also virtual.  
We probably shouldn't use the multiple inheritance trick though

Reply:
Makes sense. Let’s go with this for now.